pkg/descheduler/strategies/lownodeutilization_test.go

```diff
@@ -327,3 +333,177 @@ func TestValidateThresholds(t *testing.T) {
 		}
 	}
 }
+
+func newFake(objects ...runtime.Object) *core.Fake {
+	scheme := runtime.NewScheme()
+	codecs := serializer.NewCodecFactory(scheme)
+	fake.AddToScheme(scheme)
+	o := core.NewObjectTracker(scheme, codecs.UniversalDecoder())
+	for _, obj := range objects {
+		if err := o.Add(obj); err != nil {
+			panic(err)
+		}
+	}
+
+	fakePtr := core.Fake{}
+	fakePtr.AddReactor("list", "pods", func(action core.Action) (bool, runtime.Object, error) {
+		objs, err := o.List(
+			schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods"},
+			schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Pod"},
+			action.GetNamespace(),
+		)
+		if err != nil {
+			return true, nil, err
+		}
+
+		obj := &v1.PodList{
+			Items: []v1.Pod{},
+		}
+		for _, pod := range objs.(*v1.PodList).Items {
+			podFieldSet := fields.Set(map[string]string{
+				"spec.nodeName": pod.Spec.NodeName,
+				"status.phase":  string(pod.Status.Phase),
+			})
+			match := action.(core.ListAction).GetListRestrictions().Fields.Matches(podFieldSet)
+			if !match {
+				continue
+			}
+			obj.Items = append(obj.Items, *pod.DeepCopy())
+		}
+		return true, obj, nil
+	})
+	fakePtr.AddReactor("*", "*", core.ObjectReaction(o))
+	fakePtr.AddWatchReactor("*", core.DefaultWatchReactor(watch.NewFake(), nil))
+
+	return &fakePtr
+}
+
+func TestWithTaints(t *testing.T) {
+	strategy := api.DeschedulerStrategy{
+		Enabled: true,
+		Params: api.StrategyParameters{
+			NodeResourceUtilizationThresholds: api.NodeResourceUtilizationThresholds{
+				Thresholds: api.ResourceThresholds{
+					v1.ResourcePods: 20,
+				},
+				TargetThresholds: api.ResourceThresholds{
+					v1.ResourcePods: 70,
+				},
+			},
+		},
+	}
+
+	n1 := test.BuildTestNode("n1", 1000, 3000, 10)
```

Comment:
1000 -> 2000. Given the first test creates 8 pods all scheduled to node n1, each pod eating 200m, they all eat 1600m which is more than is available on the node. It's a bit confusing.